docs/sidebars.js

```diff
@@ -65,7 +64,6 @@ module.exports = {
         }
       ],
     },
-    "community-tools",
```

Comment:
how come you deleted community tools?

Reply:
it did not build on my machine, what is this?

Reply:
it feels like community-tools is added as an mdx, was it meant to be so?

Reply:
fixed